concent_api/conductor/tests/test_tasks.py

```diff
@@ -53,51 +53,52 @@ def setUp(self):
         blender_subtask_definition.save()
 
     def test_that_upload_acknowledged_task_should_change_upload_finished_on_existing_related_verification_request_to_true_and_call_blender_verification_order_task(self):
-        store_subtask(
-            task_id=self.report_computed_task.task_to_compute.compute_task_def['task_id'],
-            subtask_id=self.report_computed_task.task_to_compute.compute_task_def['subtask_id'],
-            provider_public_key=self.PROVIDER_PUBLIC_KEY,
-            requestor_public_key=self.REQUESTOR_PUBLIC_KEY,
-            state=Subtask.SubtaskState.ADDITIONAL_VERIFICATION,
-            next_deadline=get_current_utc_timestamp() + settings.CONCENT_MESSAGING_TIME,
-            task_to_compute=self.report_computed_task.task_to_compute,
-            report_computed_task=self.report_computed_task,
-        )
+        with freeze_time():
```

Comment:
Why is this needed now ?

Reply:
because this test fails from time to time ;) without `freeze_time` there is sometimes 1 sec difference between times